src/core/blueprint/operators/apollo_federation.rs

```diff
@@ -37,7 +37,8 @@ pub fn compile_entity_resolver(inputs: CompileEntityResolver<'_>) -> Valid<IR, S
                     config_module,
                     http,
                     // inner resolver should resolve only single instance of type, not a list
-                    false,
+                    &Default::default(),
```

Comment:
Pass Type and Args over here. Can create a new struct — `HttpParams {type_of: Option<Type>, args: Option<Args>}`  to set the HttpParams for the compiler to work.